src/OpenLoco/src/Map/Track/TrackData.cpp

```diff
@@ -331,17 +331,454 @@ namespace OpenLoco::World::TrackData
         return roadPieces[trackId];
     }
 
-    static Interop::loco_global<TrackCoordinates[80], 0x004F6F8C> _4F6F8C;
-    static Interop::loco_global<TrackCoordinates[352], 0x004F7B5C> _4F7B5C;
+    // 0x4F6F8C
+    static constexpr std::array<TrackCoordinates, 80> kRoadCoordinates = { {
+        { 0, 0, { -32, 0, 0 } },
```

Comment:
It would be good to split these up by trackId/roadId

Reply:
Could you elaborate? I don't see an obvious way to do so. I think it's indexed by TaD?

Reply:
Sounds like a good idea. I'm not sure I see where those boundaries should go, though. My thinking is when the first two members (`rotationBegin` and `rotationEnd`) are `0`? In that case, I can make 10 groups in `kRoadCoordinates`, but only 14 in `kTrackCoordinates`, which does not line up with the amount of elements and so therefore feels wrong.